src/Sylius/Component/Order/Model/OrderSequence.php

```diff
@@ -24,6 +24,11 @@ class OrderSequence implements OrderSequenceInterface
     /**
      * @var int
      */
+    protected $version = 1;
```

Comment:
I wanted to avoid the trait 😄  We don't have traits for e.g. `SlugAwareInterface` or `CodeAwareInterface` so I thought that we did't want to create another traits. What do you think @pjedrzejewski ?